src/App.vue

```diff
@@ -1,20 +1,33 @@
 <template>
   <Error />
+  <Loading :loadingAnim="showLoading"/>
```

Comment:
change loadingAnim to loading or any other better variable name

Reply:
changed